op-interop-mon/flags/flags.go

```diff
@@ -0,0 +1,66 @@
+package flags
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+)
+
+const EnvVarPrefix = "OP_INTEROP_MON"
+
+func prefixEnvVars(name string) []string {
+	return opservice.PrefixEnvVar(EnvVarPrefix, name)
+}
+
+var (
+	// Required Flags
+	L2RpcsFlag = &cli.StringSliceFlag{
+		Name:     "l2-rpcs",
+		Usage:    "The RPC URLs for the L2 chains to monitor",
+		EnvVars:  prefixEnvVars("L2_RPCS"),
+		Required: true,
+	}
+
+	// Optional Flags
+	PollIntervalFlag = &cli.DurationFlag{
+		Name:    "poll-interval",
+		Usage:   "How frequently to poll L2 for new blocks",
+		Value:   12 * time.Second,
```

Comment:
Curious why we want 12s the default? Do you think there is any value in having a different poll interval per L2?

Reply:
This value should be closer to the block production rate per chain, and the 12s was just a placeholder. Ideally this is `2` or `1`.

Per-chain settings would be cool, but as currently designed it will only identify chains through the RPC connections it makes. Maybe we want to supply the Dependency Set to the monitor? Then we would have a clear idea of what networks to expect, and could support per-chain config

Reply:
Oh also i think this isn't used, since there is just a subscription to blocks instead.

Reply:
removed.